### PR TITLE
reside-112: documentation and lint improvements

### PR DESCRIPTION
**Pull request by richfitz**
Most critically, this removes stale docs about an option that does not exist.  It also delints the package where it passes `lintr::lint_package()` - I'll see if I can configure codefactor (edit: codefactor would require at least a discussion with other ropensci folks so leaving for now)

### CODE REVIEW

File: vignettes/src/data.R [hunk 1]
```diff
@@ -163,7 +163,7 @@ head(cyphr::decrypt(readRDS(filename), key_bob))
 unlink(data_dir, recursive = TRUE)
 dir.create(data_dir)
 
-## Setup, on alice's computer computer:
+## Setup, on Alice's computer computer:
```
Comment (r-ash) [inline vignettes/src/data.R]: Dupe `computer`? Same for the line about Bob on 175

File: R/data.R [hunk 2]
```diff
@@ -227,9 +227,16 @@ data_admin_list_keys <- function(path_data = NULL) {
...
-##'   Usually this can be omitted.  Use the \code{cyphr.user.path}
-##'   global option (i.e., via \code{options()}) to set this more
-##'   conveniently.
+##'   Usually this can be omitted.  This argument is passed in as both
+##'   \code{pub} and \code{key} to \code{\link{keypair_openssl}}.
+##'   Briefly, if this argument is not given we look at the
+##'   environment variables \code{USER_PUBKEY} and \code{USER_KEY} -
+##'   if set then these must refer to path of your public and private
+##'   keys.  If these environment variables are not set then we fall
+##'   back on \\code{~/.ssh/id_rsa.pub} and \code{~/.ssh/id_rsa},
```
Comment (r-ash) [inline R/data.R]: Double `\\` on `code{~/.ssh/id_rsa.pub}`

**Review by r-ash — CHANGES_REQUESTED** [review]
Looks good, couple of typos probably worth fixing.

**Comment by richfitz** [comment]
nice spots thanks :pray:

**Review by r-ash — APPROVED** [review]

**Pull request merged**